js/html-forms_Finding-a11y-errors/index.html

```diff
@@ -11,21 +11,27 @@
   </head>
   <body>
     <section>
-      <h1 class="title">Understanding A11y in Forms</h1>
+      <h1 class="title" id="title">Understanding A11y in Forms</h1>
       <p id="description">
         Please let us know how we can support you with a11y issues.
       </p>
 
       <form for="title">
         <fieldset>
           <legend>Enter your information</legend>
-          <label aria-describedby="name-input">Name:</label>
+          <label aria-describedby="Put name here" for="name-Input">Name:</label>
           <input id="name-Input" type="text" name="name" />
```

Comment:
I made this comment on the code of other colleagues, I'm not sure whether it makes a difference or not having the attributes ID and NAME with the same value. I doesn't make any difference for the page's functionality, but maybe for accessibility. Let's ask Sven.